io.github.yairm210.unciv.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.yairm210.unciv</id>
+  <name>UnCiv</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+  <developer_name>Yair Morgenstern</developer_name>
+  <summary>Turn-based strategy game</summary>
+  <summary xml:lang="fr">Jeu de stratégie au tour par tour</summary>
+  <description>
+    <p>A reimplementation of the most famous civilization-building game ever—fast, small, no ads, free forever!</p>
+    <p>Build your civilization, research technologies, expand your cities and defeat your foes!</p>
+    <p>Requests? Bugs? Visit the todo list at https://github.com/yairm210/UnCiv/issues, every small help is welcome!</p>
+    <p>Questions? Comments? Just bored? Join us on https://discord.gg/bjrB4Xw</p>
+    <p>Want to help translating the game into your language? Send me an email to yairm210@hotmail.com.</p>
+    <p>Grok Java, Kotlin or C#? Join us at https://github.com/yairm210/UnCiv.</p>
+    <p>The world awaits! Will you build your civilization into an empire that will stand the test of time?</p>
+    <p>Internet permissions required in order to download large font files, especially for Chinese, which, if included by default, would more than double (!) the app size.</p>
```

Comment:
```suggestion
```
I don't see why a permission has to be explained in the description of the application.

Reply:
It was there in the f-droid description, I left it there in case the reader was wondering.
I personally like it a lot when an app explains this, but I can remove. I also felt this was incomplete, as the app now has multiplayer.

Reply:
It should be deleted, Flatpak (and some frontends) show requested permissions.